docs/source/tutorial/spaceflights_tutorial.md

```diff
@@ -1,59 +1,68 @@
+* use the [#questions channel](https://kedro-org.slack.com/archives/C03RKP2LW64) on our Slack channel (which replaces our Discord server) to ask the community for help
 
-### 3. Create the pipeline
+## Terminology
 
-* Create the data transformation steps as Python functions
-* Add your functions as nodes, to construct the pipeline
-* Choose how to run the pipeline: sequentially or in parallel
+We will explain any Kedro-specific terminology we use in the tutorial as we introduce it. We use additional terminology that may not be familiar to some readers, such as the concepts below.
 
-### 4. Package the project
+### Project root directory
+Also known as the "root directory", this is the parent folder for the entire project. It is the top-level folder that contains all other files and directories associated with the project.
 
- * Build the project documentation
- * Package the project for distribution
+### Dependencies
+These are Python packages or libraries that an individual project depends upon to complete a task. For example, the Spaceflights tutorial project depends on the [scikit-learn](https://scikit-learn.org/stable/) library.
 
-## Optional: Git workflow
+### Standard development workflow
+When you build a Kedro project, you will typically follow a [standard development workflow](../faq/faq.md#what-is-the-typical-kedro-project-development-workflow):
 
-### Create a project repository
+1. Set up the project template
 
-We recommend that you use `git` for source control, but Kedro does not require it, and can work without any source control management system. This section is optional if you choose not to use a `git` repository.
+    * Create a new project and install project dependencies.
+    * Configure credentials and any other sensitive/personal content, and logging
 
-```{note}
-If you are unfamiliar with a typical `git` workflow, you can follow one of the most popular, known as [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
-```
+2. Set up the data
+
+    * Add data to the `data` folder
+    * Reference all datasets for the project
+
+3. Create the pipeline
 
-If you don't have a local `git` repository for your project already, navigate to the project directory and create one:
+    * Construct nodes to make up the pipeline
+    * Choose how to run the pipeline: sequentially or in parallel
+
+4. Package the project
+    * Build the project documentation
+    * Package the project for distribution
+
+
+### Source control with `git`
+
+We recommend that you use `git` for source control. If you are unfamiliar with a typical `git` workflow but want to learn more, we suggest you look into [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
+
+Navigate to the project root directory and create a `git` repository on your machine (a local repository) for the project:
 
 ```bash
 git init
```

Comment:
When I'm doing tutorials I simply do `git init`, `git add .` and then `git commit -m "initial commit"` this is because then the IDE shows me what I've changed since compared to the templated structure provided.